apps/coscrad-frontend/src/components/resources/photographs/photograph-detail.thumbnail.presenter.tsx

```diff
@@ -0,0 +1,26 @@
+import { IDetailQueryResult, IPhotographViewModel, ResourceType } from '@coscrad/api-interfaces';
+import { routes } from 'apps/coscrad-frontend/src/app/routes/routes';
+import { Link } from 'react-router-dom';
+import './photograph-detail.thumbnail.presenter.css';
+
+export const PhotographDetailThumbnailPresenter = ({
+    data: photograph,
+}: IDetailQueryResult<IPhotographViewModel>): JSX.Element => {
+    const location = `/${routes.resources.ofType(ResourceType.photograph).detail(photograph.id)}`;
+
+    return (
+        <Link to={location}>
```

Comment:
I'd rather see an implicit return and in-line the template literal on line 9. Then you don't need to think about what to call that const.

Reply:
Can you spell this out for me?  Not quite getting it.

Reply:
Done